easybuild/easyconfigs/c/cuTENSOR/cuTENSOR-1.5.0.3-CUDAcore-11.4.2.eb

```diff
@@ -0,0 +1,30 @@
+#Author: Moise Rousseau (moise.rousseau@calculquebec.ca)
+#Date: 31st May 2022
+
+easyblock = 'Tarball'
+
+name = 'cuTENSOR'
+version = '1.5.0.3'
+
+homepage = 'https://developer.nvidia.com/cutensor'
+description = """cuTENSOR is a high-performance CUDA library for tensor 
+primitives."""
+
+toolchain = {'name': 'CUDAcore', 'version': '11.4.2'}
+toolchainopts = {'optarch': 'False'}
+
+# By downloading, you accept the cuTENSOR Software License Agreement
+# (https://docs.nvidia.com/cuda/cutensor/license.html)
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/compute/cutensor/redist/libcutensor/linux-x86_64/libcutensor-linux-x86_64-%(version)s-archive.tar.xz']
```

Comment:
Duplicate archive. You can remove the one from the urls.

Reply:
done